### ISSUE TITLE
Morph Browser chrashes when I try to save a link or image

### ISSUE THREAD

**Issue by LGeekMX** [ubports/ubuntu-touch#1733]
- Device: OnePlus One (bacon)
- Channel: devel-hallium
- Build: Ubuntu 16.04 (2021-05-20)

### Steps to reproduce
Trying to save any image or link causes Morph to crash and closes itself.

### Expected behavior
Morph shoud ask if i want to save image or link

### Actual behavior
Morph crash, the app just close.

### Logfiles and additional information
http://paste.ubuntu.com/p/Vy4Bh9YgJ3/

**Comment by kugiigi** [comment]
Did you change your scaling? If yes, does morph also crash when trying to bookmark or by tapping the lock/globe icon in the address bar?

**Comment by LGeekMX** [comment]
> Did you change your scaling? If yes, does morph also crash when trying to bookmark or by tapping the lock/globe icon in the address bar?

Yes, I have changed the scaling, also crashes when tap on bookmark button, cant test lock button because I switch to Stable channel.

**Comment by kugiigi** [comment]
I see, then it's the same weird issue in morph where certain popups crash it when scaling is set to certain values, 20 I believe.
https://github.com/ubports/morph-browser/issues/312
